content/news/2020-08-28-gsoc-ends-2020.md

```diff
@@ -0,0 +1,56 @@
+ - [add more types, making sure the macro works](https://github.com/ruma/ruma-events/pull/107)
+ - [work on `Event` custom macro derive](https://github.com/ruma/ruma-events/pull/108)
+ - [remove `raw` modules and `FromRaw` trait](https://github.com/ruma/ruma-events/pull/111)
+ - [add each events enum](https://github.com/ruma/ruma-events/pull/119)
+ - [add trybuild tests for `ruma-events`](https://github.com/ruma/ruma-events/pull/122)
+
+## Move to the mono-repo
+
+ - [deserialization optimization for Any*Event, and benchmarking to verify](https://github.com/ruma/ruma/pull/52)
+ - [reverting the removal of enum_enum macro, learning a lot of git along the way](https://github.com/ruma/ruma/pull/68)
+ - [it took me awhile but I got the hang of it, only one review issue!](https://github.com/ruma/ruma/pull/69)
+ - [add accessor methods to the enums, making them much nicer to use](https://github.com/ruma/ruma/pull/97)
+
+## Work on ruma-client-api
+
+ - [refactoring the `Api::to_tokens` code helped me get my head around this crate](https://github.com/ruma/ruma/pull/104)
+ - [notice my commit messages are starting to looking better](https://github.com/ruma/ruma/pull/105)
+ - [not everything works out...](https://github.com/ruma/ruma/pull/108)
+ - [but you get it eventually](https://github.com/ruma/ruma/pull/109)
+ - [deep dive into rust features, I learned a lot about them](https://github.com/ruma/ruma/pull/111)
+
+## Back to ruma-events
+
+ - [a ruma events and client-api rename](https://github.com/ruma/ruma/pull/130)
+ - [add code to generate `redact` method in event_enum!](https://github.com/ruma/ruma/pull/138)
+ - [adding to existing code without breaking everything is hard](https://github.com/ruma/ruma/pull/139)
+ - [converting to and from the rusty way](https://github.com/ruma/ruma/pull/155)
+ - [I love writing proc_macros, this was a fun one! Adding lifetimes in the req/resp macro](https://github.com/ruma/ruma/pull/179)
+
+## Continuing maintenance
+
+ - [quality of life stuff, the day to day work of a maintainer](https://github.com/ruma/ruma/pull/190)
+ - [allow the internal macros to be used outside of the ruma crate more easily](https://github.com/ruma/ruma/pull/201)
+ - [adding small needed features to the macros](https://github.com/ruma/ruma/pull/213)
+ - [good ol' fashioned closing issues!](https://github.com/ruma/ruma/pull/234)
+
+ One of my personal goals was to become more familiar with git. With the help of my mentor I now feel more confident using this tool that is so essential to developers. I became fairly adept at merging/rebasing, and navigating all the headaches that come with that. I learned plenty of new commands, a few highlights, `cherry-pick` and specific uses of `reset` to avoid copy/pasting fixes and adding more commits. I used the `reset` command to craft good commits, splitting work into appropriate chunks. I am glad that I had the opportunity to hone my git skills. I feel like I have accomplished my goal and then some!
```

Comment:
```suggestion
 One of my personal goals was to become more familiar with git. With the help of my mentor I now feel more confident using this tool that is so essential to developers. I became fairly adept at merging, rebasing, and navigating all the headaches that come with that. I learned plenty of new commands. A few highlights: `cherry-pick` and specific uses of `reset` to avoid copy-pasting fixes and adding more commits. I used the `reset` command to craft good commits, splitting work into appropriate chunks. I am glad that I had the opportunity to hone my git skills. I feel like I have accomplished my goal and then some!
```